CMakeLists.txt

```diff
@@ -125,7 +125,7 @@ find_package(fmt REQUIRED CONFIG)
 configure_file(source/Version.hpp.in "${PROJECT_SOURCE_DIR}/include/mrdox/Version.hpp")
 file(GLOB_RECURSE SOURCES CONFIGURE_DEPENDS source/*.cpp source/*.hpp source/*.in source/*.natvis)
 file(GLOB_RECURSE INCLUDES CONFIGURE_DEPENDS include/*.hpp)
-
+file(COPY addons DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
We should not be copying the addons directory to the location of the executable via CMake.

You should do this only when we are building the release package (i.e. the installer).